pkg/verification/runme.sh

```diff
@@ -48,13 +48,13 @@ dump() {
 
 do_help() {
 cat <<__EOT__
-Usage: docker run [-v <option>] lfedge/verification [-f <fmt>] [-p <platform>] [--accept-license] version|verification_rootfs|verification_live|verification_raw|verification_iso|verification_net
+Usage: docker run [-v <option>] lfedge/eve-verification [-f <fmt>] [-p <platform>] [--accept-license] version|verification_rootfs|verification_live|verification_raw|verification_iso|verification_net
```

Comment:
I didn't see a verification_net in the Makefile.

Reply:
This is on another PR (that is already merged).

$(VERIFICATION).net: $(EFI_PART) $(ROOTFS_IMG) $(INITRD_IMG) $(VERIFICATION_IMG) $(CONFIG_IMG) $(PERSIST_IMG) $(KERNEL_IMG) | $(VERIFICATION)
	./tools/makenet.sh $| verification.img $@
	$(QUIET): $@: Succeeded